src/i18n/locales/nl.json

```diff
@@ -1,22 +1,22 @@
 {
   "common": {
-    "blockExplorer": "Blokkeer Explorer",
+    "blockExplorer": "block Explorer",
```

Comment:
This one would need to be capitalized

Reply:
Whoops, I think a find and replace messed that one up. I checked some more and fixed a couple other capitalization mistakes as well.